### DISCUSSION TITLE
v1.97.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#7510] [Announcements]
# v1.97.0

## Highlights

Welcome to release `v1.97.0` of Immich. This release contains bug fixes and minor improvements in asset rendering on the mobile app.

- Remove the polling option for library watcher due to the number of issues it brings outweighing the advantages
- Fix the issue with the toggle button on the web app that doesn't conform to state change.
- Fix the wrong aspect ratio of the preview image in the memory view

- - - -

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="SUPPORT THE PROJECT!"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 3QVAb9dCHutquVejeNXitPqZX26Yg5kxb7

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

### 🗄️ Server
* feat(server): rkmpp hardware decoding scaling by @fyfrey in https://github.com/immich-app/immich/pull/7472
* chore: remove watcher polling option by @etnoy in https://github.com/immich-app/immich/pull/7480
### 📱 Mobile
* fix(mobile): Fixes thumbnail size with blur and alignment in video player by @martyfuhry in https://github.com/immich-app/immich/pull/7483
* fix(mobile): Fixes prefer remote assets in thumbnail provider by @martyfuhry in https://github.com/immich-app/immich/pull/7485
* feat(mobile): Adds better precaching for assets in gallery view and memory lane by @martyfuhry in https://github.com/immich-app/immich/pull/7486
### 🖥️ Web
* fix(web): keep notifications in view when scrolling by @michelheusschen in https://github.com/immich-app/immich/pull/7493
* fix(web): re-render albums by @martabal in https://github.com/immich-app/immich/pull/7403
* fix(web): settings accordion open state by @michelheusschen in https://github.com/immich-app/immich/pull/7504
### 📓 Documentation
* chore: add agpl milestone by @etnoy in https://github.com/immich-app/immich/pull/7479
### 🔨 Maintenance
* chore(deps): update dependency @types/node to v20.11.20 by @renovate in https://github.com/immich-app/immich/pull/7474
* refactor(server): format and metadata e2e by @jrasm91 in https://github.com/immich-app/immich/pull/7477
* add explicit registry to docker image names by @saimonn in https://github.com/immich-app/immich/pull/7496

## New Contributors
* @saimonn made their first contribution in https://github.com/immich-app/immich/pull/7496

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.96.0...v1.97.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.97.0'>v1.97.0</a>.</em>

**Comment by joelis10** [comment]
Not saying we don't appreciate all the work you put in, but how come such a small update bumped the version up 0.1.0 instead of 0.0.1?

**Reply by alextran1502** [reply]
Because there are feature releases in this update #7472 #7486

**Reply by joelis10** [reply]
fair enough, good to know what constitutes a new update
